src/models/request.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable strict */
+
+'use strict';
+
+const { Model } = require('objection');
+
+const fulfilmentTypeEnum = [
+  'IN_KIND_DONATION',
+  'PARTNER_REFERRAL',
+  'THIRD_PARTY_PAYMENT',
+  'CASH_TRANSFER',
+];
+
+// checklist items associated with fulfilment type
+const fulfilmentChecklistEnum = [
+  {
+    IN_KIND_DONATION: [
+      'ITEMS_PURCHASED',
+      'PURCHASE_AND_REIMBURSEMENT',
+      'REIMBURSEMENT_PAID',
+      'DELIVERED_TO_BENEFICIARY',
+    ],
+  },
+  {
+    PARTNER_REFERRAL: [
+      'REFERRED_TO_PARTNER',
+      'REFERRAL_APPROVED',
+      'DELIVERED_TO_BENEFICIARY',
+    ],
+  },
+  { THIRD_PARTY_PAYMENT: ['PURCHASE_VOUCHER', 'PAYMENT_PROCESSED'] },
+  { CASH_TRANSFER: ['PURCHASE_VOUCHER', 'PAYMENT_PROCESSED'] },
+];
+
+const requestStatusEnum = ['ACCEPTED', 'REJECTED', 'UNDER_REVIEW'];
+
+const tableRequest = 'request';
+class Request extends Model {
+  static get tableName() {
+    return tableRequest;
+  }
+
+  static get jsonAttributes() {
```

Comment:
Add a comment

Reply:
added comment "Tell objection.js not to convert any array into json object"